test/check-machines-create

```diff
@@ -1328,8 +1328,12 @@ vnc_password= "{vnc_passwd}"
                     user_login = virt_install_cmd_out.split("user-login=", 1)[1].split(",")[0].rstrip()
                     self.assertIn(user_login, self.user_login)
 
-            # HACK: wait for virt-install to finish; this only races on tumbleweed
-            if self.machine.image == 'opensuse-tumbleweed' and self.create_and_run:
+            # HACK: on some OSes we need to wait for virt-install to
+            # exit; on others, it wont ever exit but that's fine as
+            # well.  (On some OSes the created VM crashes in the
+            # kernel, on others, the BIOS can't find a bootable
+            # device. Maybe that's the difference.)
+            if self.create_and_run and self.machine.image in ['opensuse-tumbleweed', 'centos-10']:
```

Comment:
I don't have proof that this affects rhel-10-0 (or not), as the last gating run in real RHEl 10 all failed due to SELinux. But it almost surely needs the same treatment. I triggered a rhel-10-0 bots run.

Reply:
The [rhel-10-0 run](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1765-7f262a27-20240814-011914-rhel-10-0/log.html) confirms this. I suggest to write this as

    self.machine.image.startswith(('opensuse-tumbleweed', 'centos-10', 'rhel-10'))

so that it also catches future rhel-10-* images.

Reply:
I have done that, and also increased the global timeout. I think opensuse runs into it.